tests/dummy/app/services/apollo.js

```diff
@@ -0,0 +1,26 @@
+import ApolloService from 'ember-apollo-client/services/apollo';
```

Comment:
What do these changes do?

Reply:
This enables Apollo Client to more-accurately match fragment types.

https://www.apollographql.com/docs/react/data/fragments/

Out of the box, Apollo Client doesn’t analyse the schema in advance and uses a naive “heuristic fragment matcher” to figure out what to do when it comes across a fragment selection in a union type. The code in this service is how we feed Apollo Client information about union types that it might encounter. I was surprised there is so much boilerplate involved but there you have it 😅

Reply:
Thanks for the info!